starter-core/src/main/java/io/micronaut/starter/feature/agorapulse/worker/template/emailDigestDistributedJobKotlin.rocker.raw

```diff
@@ -0,0 +1,34 @@
+@import io.micronaut.starter.application.Project
+
+@args (
+Project project
+)
+
+@if (project.getPackageName() != null) {
+package @project.getPackageName()
+
+}
+
+import com.agorapulse.worker.annotation.Cron
+import com.agorapulse.worker.annotation.FixedRate
+import io.micronaut.core.async.publisher.Publishers
+import jakarta.inject.Singleton
+import org.reactivestreams.Publisher
+
+@@Singleton
+class EmailDigestDistributedJob {
+
+    internal val emailsSent: MutableList<String> = arrayListOf()
+
+    @@Cron("45 6 * * *")
+    fun generateEmailsForDigest(): Publisher<String> {
```

Comment:
Annotate  return with `@SingleResult`

Reply:
the method is supposed to return multiple items but for illustration purposes it only returns one so the feature don't have to depend on any reactive implementation